bin/oref0-setup.sh

```diff
@@ -283,35 +283,38 @@ echo
 # This section is echoing (commenting) back the options you gave it during the interactive setup script.
 # The "| tee -a /tmp/oref0-runagain.sh" part is also appending it to a file so you can run it again more easily in the future.
 
-echo "# To run again with these same options, use:" | tee /tmp/oref0-runagain.sh
-echo -n "oref0-setup --dir=$directory --serial=$serial --cgm=$CGM" | tee -a /tmp/oref0-runagain.sh
+OREF0_RUNAGAIN=`mktemp /tmp/oref0-runagain.XXXXXXXXXX`
```

Comment:
more a standard security practice than a great threat. btw with kernel 4.x and the gpio python lib it would be possible to ditch mraa and to just run under normal user privileges. i agree that `sudo` is probably still required for quite some networking commands and such.